webapp/commonComponents/hooks/useProfilePicture.js

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react'
+
+import User from '../../../common/user/user'
+
+import { useAsyncGetRequest } from '.'
+
+export default (userUuid, user) => {
```

Comment:
why both arguments? can it just take the userUuid?

Reply:
Using only userUuid now, using a key to force refresh in appHeader